tests/integration/core/remote_operations.py

```diff
@@ -937,14 +937,19 @@ def clear_ha(self, server_list):
 
             firewall = RemoteFirewallControl.create(address, self._ssh_address_no_check)
 
-            # clear_ha_el7.sh
-            result = self._ssh_address(address, "if crm_mon -b1; then crm_resource -l|xargs pcs resource disable; fi")
+            result = self._ssh_address(address, "if crm_mon -b1; then pcs property set maintenance-mode=true; fi")
+            logger.debug("CMD OUTPUT:\n%s" % result.stdout)
+
+            result = self._ssh_address(address, "if crm_mon -b1; then crm_resource -l | xargs pcs resource disable; fi")
```

Comment:
`crm_resource -l | xargs -n 1 crm_resource --set-parameter target-role --meta --parameter-value Started --resource`

Reply:
hrrm, I think this should be `--parameter-value Disabled`?

Reply:
meh, `Stopped` works since we are in maintenance-mode.